src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private Ball ball = new Ball();
```

Comment:
Avoid using static methods or fields for mutable objects. It's better to create a new instance of `Ball` inside the `getRandomBall` method to ensure that a new `Ball` object is created every time the method is called.